DEVELOPING.md

```diff
@@ -12,24 +12,19 @@ Our development setup assumes a LINUX/BSD environemnt.
 6. Add the upstream repo to your remotes `git remote add upstream git@github.com:box/box-ui-elements.git`.
 7. Verify your remotes are properly set up `git remote -v`. You should pull updates from the `upstream` box repo and push changes to your fork `origin`.
 
-## Testing UI Elements with demo data
+## Examples page (aka style guide)
 
-Examples can be found in the [examples folder](examples). These examples use demo data within the UI Elements. Due to security restrictions, the data is read-only.
+A published version of the style guide can be seen [here](https://opensource.box.com/box-ui-elements/). The style guide uses **_live_** demo data within the UI Elements. Due to security restrictions this data is read-only. You can run a local version of the style guide as follows:
 
-1. Start the examples/styleguide server via `yarn start`.
-2. Navigate to `http://localhost:6060/` to see the UI Elements in action.
+1. Start the style guide server via `yarn start` (or via `yarn start:ie11`, see the **NOTES** below).
+2. Navigate to `http://localhost:6060/` to see the UI Elements in action. If testing on a different machine or VM, you can instead use the IP address shown on your terminal window.
 
-## Testing UI Elements with your data
+**_NOTE:_** _To simplify debugging, by default the style guide running on your local machine assumes the latest and greatest Chrome browser. If instead you want IE11 support, use `yarn start:ie11`._
```

Comment:
Seems like this could be shortened and inlined with step one:
> Start the style guide server via `yarn start` (or via `yarn start:ie11` to include support for IE 11)